metron-deployment/roles/quick_dev/tasks/main.yml

```diff
@@ -15,23 +15,50 @@
 #  limitations under the License.
 #
 ---
-#
-# Workaround for Kafka not starting
-# Fire off async start followed by
-# Sync start -execution will pause until
-# final start completes.
-#
-- name: Start the ambari cluster - no wait
-  ambari_cluster_state:
+- name: Delete the Metron Components from Ambari
```

Comment:
Ah, ok.  So our Atlas image has Metron pre-installed via the MPack.  But this role then uninstalls and reinstalls it so that we get the latest, right?